provider/k8s/controller_deployment.go

```diff
@@ -154,9 +156,11 @@ func (c *DeployController) SyncPDB(d *apps.Deployment, remove bool) error {
 		remove = true
 	}
 
-	minAvailable := &intstr.IntOrString{
+	c.Provider.PdbDefaultMinAvailablePercentage = common.CoalesceString(os.Getenv("PDB_DEFAULT_MIN_AVAILABLE_PERCENTAGE"),"50")
```

Comment:
do not initialize the var here. you should initialize that in here: https://github.com/convox/convox/blob/master/provider/k8s/k8s.go#L94